### ISSUE TITLE
[Bug]: react-tsparticles npm package not using correct engine dependency

### ISSUE THREAD

**Issue by Vija02** [tsparticles/tsparticles#4577]
### Contact Details

hi@michaelsalim.co.uk

### What happened?

In `react-tsparticles` v2.1.4, its `package.json` has a dependency of `tsparticles-engine` for v2.0.6. 
Looking at the repo, the version looks to be matching: https://github.com/matteobruni/tsparticles/blob/main/components/react/package.json

However, if I pull the package from npm, it says otherwise.

Steps to reproduce:
```
yarn add react-particles
cat node_modules/react-tsparticles/package.json
```

Here's the full `package.json` 
```json
{
  "name": "react-tsparticles",
  "version": "2.1.4",
  "description": "Official tsParticles React Component - Easily create highly customizable particle, confetti and fire>
  "scripts": {
    "install": "node ./scripts/install.js"
  },
  "repository": {
    "url": "https://github.com/matteobruni/tsparticles",
    "directory": "components/react",
    "type": "git"
  },
  "keywords": [
    "tsparticles",
    "particles.js",
    "particlesjs",
    "particles",
    "particle",
    "canvas",
    "jsparticles",
    "xparticles",
    "particles-js",
    "particles-bg",
    "particles-bg-vue",
    "particles-ts",
    "particles.ts",
    "react-particles-js",
    "react-particles.js",
    "react-particles",
    "react",
    "reactjs",
    "vue-particles",
    "ngx-particles",
    "angular-particles",
    "particleground",
    "vue",
    "vuejs",
    "preact",
    "preactjs",
    "jquery",
    "angularjs",
    "angular",
    "typescript",
    "javascript",
    "animation",
    "web",
    "html5",
    "web-design",
    "webdesign",
    "css",
    "html",
    "css3",
    "animated",
    "background",
    "confetti",
    "canvas",
    "fireworks",
    "fireworks-js",
    "confetti-js",
    "confettijs",
    "fireworksjs",
    "canvas-confetti"
  ],
  "funding": [
    {
      "type": "github",
      "url": "https://github.com/sponsors/matteobruni"
    },
    {
      "type": "buymeacoffee",
      "url": "https://www.buymeacoffee.com/matteobruni"
    }
  ],
  "bugs": {
    "url": "https://github.com/matteobruni/tsparticles/issues"
  },
  "author": "Matteo Bruni <matteo.bruni@me.com>",
  "license": "MIT",
  "main": "cjs/index.js",
  "module": "esm/index.js",
  "types": "types/index.d.ts",
  "unpkg": "particles.min.js",
  "jsdelivr": "particles.min.js",
  "peerDependencies": {
    "react": ">=16"
  },
  "dependencies": {
    "fast-deep-equal": "^3.1.3",
    "tsparticles-engine": "^2.0.6"
  }
}
```

### tsParticles Version

2.1.4

### Which library are you using?

React.js (react-particles, react-tsparticles, react-particles-js)

### tsParticles Configuration

```json
-
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matteobruni** [comment]
I've already noticed that, thanks for reporting that. It will be fixed in 2.2.0, it should install the latest version since the `^` semver is used.

**Comment by Vija02** [comment]
Awesome. It works fine for new installs but if you're upgrading (like me), `yarn.lock` likes to keep the older version so you have to update it manually.
